norma-portuguesa-405.csl

```diff
@@ -309,8 +309,8 @@
           <date-part name="month" form="short" suffix=". " text-case="lowercase"/>
```

Comment:
while we're at it, the parentheses should really be set as a prefix/suffix on the ```date``` in case any of the ```date-part``` are missing.